.github/workflows/conformance.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         validateMapping: true
         cmd: "'functions-framework --source tests/conformance/main.py --target write_legacy_event --signature-type event'"
 
-    - name: Run cloudevent conformance tests
+    - name: Run cloud event conformance tests
```

Comment:
The name for CloudEvents is still `CloudEvents` when not used as an identifier and referring to the spec.

Here and below.